pkg/instancetype/instancetype.go

```diff
@@ -357,11 +365,70 @@ func getInstancetypeApiSpec(obj runtime.Object) (interface{}, error) {
 	}
 }
 
+func checkCPUPreferenceRequirements(instancetypeSpec *instancetypev1alpha2.VirtualMachineInstancetypeSpec, preferenceSpec *instancetypev1alpha2.VirtualMachinePreferenceSpec, vmiSpec *virtv1.VirtualMachineInstanceSpec) (*k8sfield.Path, error) {
+	if instancetypeSpec != nil && instancetypeSpec.CPU.Guest < preferenceSpec.Requirements.CPU.Guest {
+		return k8sfield.NewPath("spec", "instancetype"), fmt.Errorf(insufficientInstanceTypeCPUResourcesErrorFmt, instancetypeSpec.CPU.Guest, preferenceSpec.Requirements.CPU.Guest)
+	}
+	if instancetypeSpec == nil && vmiSpec.Domain.CPU != nil {
+		// FIXME - Move PreferredCPUTopology to a string pointer and default in mutation webhook before removing this duplicate defaulting
```

Comment:
can you fix it now? Because after merge it will become permanent fixme :D

Reply:
I'll address it in a follow up PR before this merges, I didn't want to fix it as part of this series as it's going to touch various areas unrelated to this feature IMHO.